frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.css

```diff
@@ -5,12 +5,61 @@
 
 .diagrams {
   height: calc(80vh - 170px);
-  width: 100%;
   word-wrap: break-word;
 }
 
+.diagram-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: calc(100% / 3 - 2 * 10px - 2 * 16px); /* 3 rows - margin - padding */
+}
+
+.uuid {
+  color: grey;
+}
+
+.loader {
+  width: calc(100% / 3 - 2 * 10px);
+  margin: 10px;
```

Comment:
Material Design is using a 8px base grid. It's better to stick with 8px increments (or 4px).

See also the other cases where a 5px grid is used.

Reply:
Chrome says a 10px margin is used for mat-cards: 
![image](https://user-images.githubusercontent.com/23395732/228808292-4e2bcff4-b0b7-464c-a007-c4eddae51b39.png)